install/roles/docker/tasks/main.yml

```diff
@@ -69,13 +89,22 @@
   become: true
   when: is_rhel_centos_6
 
-- name: Install libselinux-python
+- name: Install libselinux-python on RHEL 6
   yum:
     name: libselinux-python
     state: present
   become: true
   when:
-    - is_rhel_centos_6 or is_fedora
+    - is_rhel_centos_6
+    - selinux_on
+
+- name: Install libselinux-python on Fedora
```

Comment:
Small nitpick: some places say`on RHEL 6` some places just have `RHEL6`, `Fedora`, etc. We should probably make these uniform possibly like: ``` - name: Install libselinux-python (Fedora) ```. We can do this at a later time because I'm sure this inconsistency extends beyond your PR.

Reply:
Let's do this on master.  I think we should normalize all the role names.

Reply:
See https://github.com/quipucords/quipucords/issues/1453